### PR TITLE
feat: added table sorting

### LINKED ISSUE #453: feat: add new feature to the table component
Requirements:
-The table header should be fixed in the top
-The user should  resize the columns
-The user should make the soring in the table

### PR DESCRIPTION
**Pull request by AlejandroYanes**
fixes #453

### CODE REVIEW

**Comment by TahimiLeonBravo** [comment]
follow the sorting styles when active and when hover like the design
https://zpl.io/blPO9Kz

File: src/components/Table/head/index.js [hunk 1]
```diff
@@ -9,26 +9,47 @@ export default function Head(props) {
...
     if (columns) {
         return columns.map((column, index) => {
-            const { header, sortable, width } = column;
+            const { header, sortable, width, defaultWidth } = column;
             const isSelected = index === selectedColumn;
+            const innerWidth = columnsWidths[index];
             return (
                 <Header
-                    key={uniqueId('header')}
+                    key={`header-${index}`}
```
Comment (LeandroTorresSicilia) [inline src/components/Table/head/index.js]: put this in a variable:
```const key = `header-${index}`;```
also in this way you can remove the eslint exception a the top of the file

File: src/components/Table/index.js [hunk 2]
```diff
@@ -74,9 +131,10 @@ Table.propTypes = {
     data: PropTypes.arrayOf(Object).isRequired,
     /**
      * Specifies the sorting direction, valid options are 'asc' or 'desc'.
-     * @ignore
      */
     sortDirection: PropTypes.oneOf(['asc', 'desc']),
+    /** Action triggered when a column header is clicked */
```
Comment (LeandroTorresSicilia) [inline src/components/Table/index.js]: Action triggered when a column is sorted. Receive the event object, field and sortDirection.

File: src/components/Table/readme.md [hunk 3]
```diff
@@ -61,3 +61,112 @@
...
+            let newSortDirection;
+            if (field === sortedBy) {
+                if (sortDirection === 'asc') {
+                    newSortDirection = 'desc';
+                } else {
+                    newSortDirection = 'asc';
+                }
+            } else {
+                newSortDirection = 'asc';
+            }
```
Comment (LeandroTorresSicilia) [inline src/components/Table/readme.md]: when implement inside Table remove this logic here

**Pull request merged**